### DISCUSSION TITLE
如何部署到有公网ip的服务器上

### DISCUSSION THREAD

**Discussion by DangoPie** [1715173329/UnblockNeteaseMusic#663] [Q&A] [answered]
不是很会docker，的大佬们看一下指令哪里有问题（其中QQ_COOKIE已配置好）
docker run -e ENABLE_FLAC=true -e ENABLE_LOCAL_VIP=true -e QQ_COOKIE="uin=;qm_keyst="" --restart=always pan93412/unblock-netease-music-enhanced -o qq kugou kuwo migu ytdlp bilibili pyncmd -p 18080:8080
docker ps命令查看后发现端口没有映射，也无法使用

**Comment by RWDai** [comment, marked as answer]
需要加-p 主机端口:容器端口 在镜像 pan93412/unblock-netease-music-enhanced前面
